hopsworks-web/yo/app/scripts/services/ElasticService.js

```diff
@@ -54,6 +54,18 @@ angular.module('hopsWorksApp')
                 globalSearch: function (searchTerm) {
                   return $http.get('/api/elastic/globalsearch/' + searchTerm);
                 },
+                /**
+                 * Do a global search hitting featurestore index
+                 */
+                globalfeaturestore: function (searchTerm) {
+                  return $http.get('/api/elastic/globalfeaturestore/' + searchTerm);
+                },
+                /**
+                 * Search under a project hitting featurestore index
+                 */
+                localFeaturestoreSearch: function (projectId, searchTerm) {
+                  return $http.get('/api/elastic/localFeaturestoreSearch/' + projectId + '/' + searchTerm);
```

Comment:
this endpoint is wrong - should be 
return $http.get('/api/elastic/localfeaturestore/' + projectId + '/' + searchTerm);